launcher/helper.h

```diff
@@ -10,9 +10,11 @@
 #pragma once
 
 class QObject;
+class QString;
```

Comment:
you're using QString as a fully defined type and not as a reference/pointer, hence forward declaration is incorrect - include the header instead